All EIPIP Meetings/Meeting 064.md

```diff
@@ -0,0 +1,331 @@
+
+**Sam**
+Oh yeah, so the previous pull request added an adoptable field and so the adoptable field is kind of like a signal to editors that says if the author isn't responsive, we are allowed or encouraged to assign it to a new author at our discretion. It's not my proposal. I kind of on the fence about it like I am with a lot of these things, I think it's a little bit extra to add this field so you can just make a new EIP and you know make yourself the author on it, where on the other hand it's nice to be able to like ensure some continuity between EIP numbers, and yeah so you know I'm a little torn on it and it was my bad for merging the first pr without getting rough consensus among editors so I reverted it and opened this new one to discuss. Since Pandapip isn't here, I'm not sure if we'll actually be able to debate very well. 
+
+**Pooja**
+That is right. I totally understand that. I think in general like it's my understanding that in general we can or we should keep any discussion open for at least two EIPIP meetings so we get to share the information with the EIP editors because sometimes some of the editors may miss their discussion. So yeah, Matt, if you have anything to add on this particular proposal, the adoptable one or in general you want to talk anything about the process?
+
+**Lightclient**
+I just think that we need to leave things open for a fairly long time if editors disagree, definitely for a couple EIP meetings, we should discuss it synchronously if there's disagreement before merging and I mean just generally waiting for, you know, mostly consensus among the editors, or at least the editors who are active, and I don't think that we should have the adoptable field so, it's my take. 
+
+**Pooja**
+Yeah one other thing that I would like to make a mention of generally in case of any changes to EIP-1 earlier we were having like two editors’ approval, but now we have like around five editors available, so it will be nice to have minimum three editors on board and get that approval done. I'm not sure if this is already added in the bot and it is taken care of, but if not, I think that would be a good idea.
+
+**Lightclient**
+There's also like a general courtesy I think that we haven't merged too many things in the past where editors disagreed even if we had two editors, unless the editor who disagreed said I’m not going to block this, like there is, does seem to be a bit of veto power amongst single editors if they think a change is bad, I don't want to like encode that but I think that it's like a reasonable courtesy to not just merge things after two, even if somebody has already said they're pretty against it.
+
+**Sam**
+Yeah, that was entirely my bad, like it's entirely my fault, like Pandapip has been adding EIPs to a manual merge-like list because some of them require it and I didn't realize that we didn't have consensus on this one so that's entirely my bad. 
```

Comment:
FYI @SamWilsn - we *did* have consensus on it on EIPIP meeting 62. It's just that 3 of the editors were absetnt, a fact that clearly both of us forgot.